lib/parzival_web/live/tools/announcement_live/show.ex

```diff
@@ -14,7 +14,7 @@ defmodule ParzivalWeb.AnnouncementLive.Show do
      socket
      |> assign(:current_page, :tools)
      |> assign(:page_title, page_title(socket.assigns.live_action))
-     |> assign(:announcement, Tools.get_announcement!(id))}
+     |> assign(:announcement, Tools.get_announcement!(id, preloads: [:author]))}
```

Comment:
Since you will need to change the get_announcement! function this call has also to be changed to 
`get_announcement!(id, [:author])`